src/components/GetStartedPopover/GetStartedPopover.tsx

```diff
@@ -114,6 +120,11 @@ export const GetStartedPopover = (platform) => {
         </Popover.Trigger>
         <Popover.List ariaLabel="Getting started guides for other platforms">
           {getStartedLinks.map((link, index) => {
+            link.href.pathname =
+              isGen1 && link.href.pathname.indexOf('gen1') < 0
```

Comment:
This is a good example of where it kinda makes sense to just have a shared util, since you're having to check the link for gen1 anyway

Reply:
thanks Scott, will do that now then!